src/ui/linkformview.js

```diff
@@ -77,6 +84,31 @@ export default class LinkFormView extends View {
 		 */
 		this.cancelButtonView = this._createButton( t( 'Cancel' ), cancelIcon, 'ck-button-cancel', 'cancel' );
 
+		/**
+		 * Keeps reference to {@link module:link/linkcommand~LinkCommand#customAttributes}.
+		 *
+		 * @readonly
+		 * @type {model:utils/collection~Collection}
```

Comment:
Collection of what?

Reply:
Why is this property public BTW? It's used internally only.

Reply:
I'd went even further and remove it completely.